pkgs/by-name/go/goshs/package.nix

```
@@ -0,0 +1,32 @@
+{
+  buildGoModule,
+  fetchFromGitHub,
+  lib,
+}:
+
+buildGoModule rec {
+  pname = "goshs";
+  version = "1.0.1";
+
+  src = fetchFromGitHub {
+    owner = "patrickhener";
+    repo = "goshs";
+    rev = "v${version}";
+    hash = "sha256-J74nUH3iBYDytXCswIexvShG5+zJEDX19saJ3ftLYhI=";
+  };
+
+  vendorHash = "sha256-6UaKh2UUyGqOriAaMhiEFr20R4W/ZWxQXtXnm/BVHOc=";
+
+  ldflags = [
+    "-s"
+    "-w"
+  ];
+
+  meta = with lib; {
+    description = "A SimpleHTTPServer written in Go, enhanced with features and with a nice design";
```

```suggestion
    description = "Simple, yet feature-rich web server written in Go";
```
Suggest not mentioning `SimpleHTTPServer` since it's not available in Python 3. Optionally include the keyword in a [`longDescription`](https://github.com/NixOS/nixpkgs/blob/1c6e20d41d6a9c1d737945962160e8571df55daa/pkgs/by-name/sw/swego/package.nix#L32-L35)